File: GameLibrary/test/cc/lib/reflector/ListReflector.java
package cc.lib.reflector;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Chris Caron on 12/1/23.
 */
public class ListReflector extends Reflector<ListReflector> {

    static {
        addAllFields(ListReflector.class);
    }

    List<Integer> intList = new ArrayList<>();

}

File: GameLibrary/test/cc/lib/kreflector/ListReflector.kt
package cc.lib.kreflector

/**
 * Created by Chris Caron on 12/1/23.
 */
class ListReflector : Reflector<ListReflector>() {
	var intList: MutableList<Int> = ArrayList()

	companion object {
		init {
			addAllFields(ListReflector::class.java)
		}
	}
}